examples/gitserver/hooks/detect-language

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# detect-language returns a string indicating the image repository to use as a base
```

Comment:
I tried it, but linguist doesn't really give us useful behavior, because it won't help us detect version (or at least, I could not find a way to make that happen).  We also can't really deal with 50% python / 50% ruby like linguist would give us.  I think linguist was a false hope on my part.

----- Original Message -----
> > @@ -0,0 +1,48 @@
> > +#!/bin/bash
> > +#
> > +# detect-language returns a string indicating the image repository to use
> > as a base
> 
> I think, at some point we should build a small Docker image we will use for
> code detection based on https://github.com/github/linguist
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1806/files#r29652534